### PR TITLE
Add rabbitmq example config

### PR DESCRIPTION
**Pull request by jsirianni**
### Proposed Change

Added RabbitMQ + Google Cloud example configuration.

##### Checklist
- [x] Changes are tested
- [x] CI has passed

### CODE REVIEW

File: config/google_cloud_exporter/rabbitmq/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# RabbitMQ Metrics with Google Cloud
+
+The RabbitMQ Receiver can be used to send RabbitMQ metrics to Google Cloud Monitoring. See the [documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/rabbitmqreceiver/README.m) for more details.
```
[the remark this answers is not in the document]
Comment (Mrod1598) [reply config/google_cloud_exporter/rabbitmq/README.md:3]: that link is broken too ^

[the remark this answers is not in the document]
Comment (jsirianni) [reply config/google_cloud_exporter/rabbitmq/README.md:3]: I think I will just link to https://github.com/observIQ/observiq-otel-collector/blob/main/docs/receivers.md, so we can use that page as the single source of truth.

Thoughts?

File: config/google_cloud_exporter/rabbitmq/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Run the following command
+```bash
+sudo systemctl edit observiq-otel-collector
+```
+
+If this is the first time an override is being created, the file will be empty. Paste the following contents into the file. If the `Service` section is already present, append the two `Environment` lines to the `Service` section.
+```
+[Service]
+Environment=RABBITMQ_USERNAME=otel
```
[the remark this answers is not in the document]
Comment (jsirianni) [reply config/google_cloud_exporter/rabbitmq/README.md:29]: Yes, done.

File: config/google_cloud_exporter/rabbitmq/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (cpheps) [reply config/google_cloud_exporter/rabbitmq/README.md:3]: I'm cool with that.

[the remark this answers is not in the document]
Comment (jsirianni) [reply config/google_cloud_exporter/rabbitmq/README.md]: @cpheps @Mrod1598 I pushed a link to our doc, which has links to the exact component version. Once this doc is merged, I will do the same to the other example PRs. Once those are merged, I will update the existing docs to reflect this standard.

**Review by cpheps — APPROVED** [review]

**Review by BinaryFissionGames — APPROVED** [review]

**Pull request merged**